tests/cases/probe_without_clear_command.expect_fail.amber

```diff
@@ -27,8 +27,8 @@ main()
 }
 
 [test]
-clear color 0.1 0.2 0.3 0.4
-
-# No clear command before probe. Random pixels are expected.
-
 relative probe rect rgba (0.0, 0.0, 1.0, 1.0) (0.1, 0.2, 0.3, 0.4)
+
+# Expected result: assert failure with the following message
```

Comment:
Can you file a bug, this might be an interesting thing to add to the test harness so we can verify that this was the output.

Reply:
Opened a bug as #60.